samtranslator/model/api/api_generator.py

```diff
@@ -1172,7 +1175,6 @@ def _set_default_authorizer(  # type: ignore[no-untyped-def]
                 default_authorizer,
                 authorizers=authorizers,
                 add_default_auth_to_preflight=add_default_auth_to_preflight,
-                api_authorizers=api_authorizers,
```

Comment:
No need to pass `api_authorizers`. `authorizers` already contains everything from `api_authorizers`. 
https://github.com/aws/serverless-application-model/blob/develop/samtranslator/model/api/api_generator.py#L723